### PR TITLE
I believe I have met the requirement

### PR DESCRIPTION
**Pull request by jmarcusgill**
I think the essential elements are in place. I would like to have styled it better, but it I think it is okay for the task at hand.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    </div>
+
+
+
+  </article>
+
+  <article class="projects">
+    <h3>Future Projects</h3>
+
+    <section class="section1"></section>
```
Comment (morecallan) [inline index.html]: For both this and the section tag below it, they should wrap the project content.

**Review by morecallan — COMMENTED** [review]
Hey @jmarcusgill, lookin' grood! 

-Make sure you add comments to your code. The project requirements are a good place to start for structure. 
-COMMIT MOREEEEEEEEEEEEEEEEEEEE

**Comment by jmarcusgill** [comment]
@morecallan,

i fixed the <section></section> issues and cleaned it up a bit. Added comments as well. Let me know if this is looking better!

**Comment by morecallan** [comment]
V. Nice! 👍

**Pull request merged**